templates/typescript_gapic/package.json.njk

```diff
@@ -27,7 +27,7 @@ limitations under the License.
     "build/protos"
   ],
   "dependencies": {
-    "google-gax": "^1.7.5"
+    "google-gax": "git://github.com:googleapis/gax-nodejs.git#export_apiTypes"
```

Comment:
Let's not merge it until we have this in gax.

Reply:
yes sure, will hold this until new release.